hi-ml-azure/run_pytest.py

```diff
@@ -107,6 +171,7 @@ def run_pytest(folder_to_test: str, pytest_mark: str) -> None:
                 snapshot_root_directory=git_repo_root_folder(),
                 conda_environment_file=config.conda_env,
                 experiment_name=config.experiment,
-                max_run_duration=config.max_run_duration
+                max_run_duration=config.max_run_duration,
+                after_submission=pytest_after_submission_hook
```

Comment:
This is a good example of why it's a good idea to add a trailing comma after the last argument. So I suggest adding a comma after "hook".